apps/flame-defi/app/pool/modules/create-position/components/fee-tier-select/fee-tier-select.tsx

```diff
@@ -0,0 +1,63 @@
+import { CheckCircleIcon } from "@repo/ui/icons";
+import { cn } from "@repo/ui/utils";
+import { FEE_TIER, FEE_TIERS, type FeeTier } from "pool/constants";
+import { useIntl } from "react-intl";
+
+const DATA: { [key in FeeTier]: { label: string } } = {
+  [FEE_TIER.LOWEST]: {
+    label: "Best for very stable pairs.",
+  },
+  [FEE_TIER.LOW]: {
+    label: "Best for stable pairs.",
+  },
+  [FEE_TIER.MEDIUM]: {
+    label: "Best for most pairs.",
+  },
+  [FEE_TIER.HIGH]: {
+    label: "Best for exotic pairs.",
+  },
+};
+
+interface FeeTierSelectProps {
+  value: FeeTier;
+  onChange: (value: FeeTier) => void;
+}
+
+export const FeeTierSelect = ({ value, onChange }: FeeTierSelectProps) => {
+  const { formatNumber } = useIntl();
+
+  return (
+    <div className="flex flex-col gap-1">
+      {FEE_TIERS.map((it) => (
```

Comment:
```suggestion
      {FEE_TIERS.map((feeTier) => (
```

Reply:
Prefer extra explicit. I think it's so helpful when grokking the value in the nested jsx